Tasks/code-push-vsts-task/CodePush.js

```diff
@@ -1,5 +1,5 @@
 var path = require("path");
-var tl = require("vso-task-lib");
```

Comment:
Doh! Sorry I clicked the wrong line :) I was referring to the directory that includes the release task. It might make sense to rename it to "codepush-release" for cleanliness sake.